format/FormatXTCJungfrau.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, image_file, **kwargs):
         FormatXTC.__init__(
             self, image_file, locator_scope=jungfrau_locator_scope, **kwargs
         )
-        self._ds = FormatXTCJungfrau._get_datasource(image_file, self.params)
+        self._ds = self._get_datasource(image_file, self.params)
```

Comment:
_get_datasource is a static function. Would prefer  leaving it the way it was because it is much easier to understand that it is a static function when reading code.

Reply:
...but then the function is not overloadable and therefore the class not subclassable in that respect? This also means that it is not clear to the reader that `FormatXTCJungfrau._get_datasource` and `FormatXTCJungfrauMonolithic._get_datasource` are in fact the _same_ function.

I am also not sure why the user of a class would care whether a function on a class is static or not?

Reply:
The function isn't even declared on `FormatXTCJungfrau`, so this only serves to misdirect anyone reading the code who would then look for it in that class?!

Reply:
@Anthchirp - The static function is defined in FormatXTC so at a bare minimum please revert it to that . I use these 2 classes on a regular basis and it affects my readability. As stated earlier, I am OK with the main premise of this PR which is removing de-duplication of code. Thanks for your attention.